Capstone-Projects-2022-Spring/project-uniflea
# Added unit test for email validation

Added unit test for email validation, and centered submit button on sign up/sign in screens.

To test:
- run yarn test, this should display tests that have passed.
- preview sign in/sign up screens to confirm the buttons are centered.
